api/anilist/build.gradle.kts

```diff
@@ -28,14 +26,15 @@ kotlin {
 }
 
 dependencies {
+    implementation(project(":api:preferences"))
```

Comment:
No type-safe project accessors?

Reply:
Imma do this in the next PR after this is merged 😎